scripts/alias_index.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+# This script will get all the ElasticSearch indexes that uses a given alias, the alias will
+# be removed from those indexes, before the index name that is passed into this script
+# is aliased using the given alias.
+# e.g. sh ./scripts/alias_index.sh localhost 9200 bi-dev-01082018 bi-dev
+# - The indexes aliased to 'bi-dev' will have their aliases removed (if any exist).
+# - The passed in index name will be aliased using the given alias ('bi-dev').
+# https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-aliases.html#indices-aliases
+
+SCRIPT_NAME=$0
```

Comment:
For portability use `${BASH_SOURCE[0]}` if you refer to current file, even if it is sourced by a parent script. In other cases, use `${0}`.